core/src/main/resources/hudson/model/AbstractBuild/index_ru.properties

```diff
@@ -21,7 +21,7 @@
 # THE SOFTWARE.
 
 on=\u043d\u0430
-startedAgo=\u0417\u0430\u043f\u0443\u0449\u0435\u043d\u0430 {0} \u0434\u043d\u0435\u0439 \u043d\u0430\u0437\u0430\u0434.
+startedAgo=\u0417\u0430\u043F\u0443\u0449\u0435\u043D\u0430 {0} \u043D\u0430\u0437\u0430\u0434
```

Comment:
It may be not only days, but also minutes, seconds and hours. I doubt this is a correct fix

Reply:
Yes, this field may contains days, hours, etc., and this is the reason that i remove the word "\u0434\u043d\u0435\u0439" from the pattern.